src/utils/invertIndexing.js

```diff
@@ -0,0 +1,47 @@
+const Keyword = require("../models/Keyword");
+const OriginalKeyword = require("../models/OriginalKeyword");
+const analyzeText = require("./analyzeText");
+const stemWord = require("./stemmer");
+
+async function invertIndexing(video) {
```

Comment:
함수 명이 동명사 형태 같습니다. 동사 형태인 `invertIndex`로 변경하면 조금 더 직관적일 것 같은데 어떠신가요?
```suggestion
async function invertIndex(video) {
```

Reply:
말씀하신 대로 수정하도록 하겠습니다!